presto-main/src/main/java/com/facebook/presto/GroupByHashPageIndexer.java

```diff
@@ -35,8 +35,10 @@ public GroupByHashPageIndexer(List<? extends Type> hashTypes)
     {
```

Comment:
The commit message needs more details about how this semijoin + indeterminate works. You might want to elaborate a bit on what "add a
new method indeterminateContains to handle indeterminate rows" actually means.